db/schema.rb

```diff
@@ -181,11 +181,15 @@
   end
 
   create_table "ratings", id: :serial, force: :cascade do |t|
-    t.integer "rating"
```

Comment:
In contrast to the categories listed below, it looks like `NULL` is allowed. Is that the case actually?

Reply:
That's right, I added a not-null constraint to be consistent with the other columns.